pkg/netpol/connlist/connlist.go

```diff
@@ -173,16 +173,23 @@ func (ca *ConnlistAnalyzer) ConnlistFromYAMLManifests(manifests []scan.YAMLDocum
 
 func (ca *ConnlistAnalyzer) connslistFromParsedResources(objectsList []scan.K8sObject) ([]Peer2PeerConnection, error) {
 	// TODO: do we need logger in policyEngine?
-	pe, err := eval.NewPolicyEngineWithObjects(objectsList)
-	if err != nil {
-		return nil, err
+	pe, evalErrs := eval.NewPolicyEngineWithObjects(objectsList, ca.stopOnError)
+	for i := range evalErrs {
+		ca.errors = append(ca.errors, &evalErrs[i])
+	}
+
+	if ca.stopProcessing() {
+		if err := ca.hasFatalError(); err != nil {
+			return nil, err
+		}
+		return []Peer2PeerConnection{}, nil
 	}
 	return ca.getConnectionsList(pe)
 }
 
 // ConnlistFromK8sCluster returns the allowed connections list from k8s cluster resources
 func (ca *ConnlistAnalyzer) ConnlistFromK8sCluster(clientset *kubernetes.Clientset) ([]Peer2PeerConnection, error) {
```

Comment:
errors returning from `getConnectionsList`, are now handled differently, i.e returned only when `ca.stopProcessing()` is true.
 this may affect this function's result ,
 should do changes of handling errors here too?
or send an indication when `getConnectionsList` is called from here?

Reply:
not relevant anymore